src/implementations/vanilla/components/banner/_banner.scss

```diff
@@ -4,24 +4,17 @@
  */
 
 @use 'sass:map';
-@use '@ecl/theme-dev/theme';
 @use '@ecl/vanilla-layout-grid/mixins/breakpoints';
 
-$_border-radius: null !default;
-$_content-width: 45.5rem;
-$_box-shadow: null !default;
-$_border-color: map.get(theme.$color, 'yellow-100');
-$_border-width: null !default;
-$_background: null !default;
-$_overlay: null !default;
-$_text-color: null !default;
-$_font: null !default;
-$_font-weight: null !default;
+// Exposed variables
+$theme: null !default;
+$banner: null !default;
 
 .ecl-banner {
   align-items: center;
-  border-radius: map.get($_border-radius, 'banner');
-  box-shadow: $_box-shadow;
+  background: var(--ecl-color-branding);
```

Comment:
not sure why we are using the 'branding' color here. As far as I know, it was meant to be used only in the header/footer.
I guess that it comes from the specs. I'll ask the designers...

Reply:
yep, this is what we have in figma: background: var(--others-branding, #254ACB);